src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/pages/Application.vue

```diff
@@ -17,7 +17,20 @@
           </v-col>
           <v-col cols="auto">
             <v-btn rounded="lg" variant="outlined" color="primary" class="mr-4" primary @click="handleSaveAsDraft">Save as Draft</v-btn>
-            <v-btn type="submit" :form="getFormId" rounded="lg" color="primary" :disabled="isDisabled" @click="handleSaveAndContinue">Save and Continue</v-btn>
+            <v-btn
+              v-if="wizardStore.step !== 7"
+              type="submit"
+              :form="getFormId"
+              rounded="lg"
+              color="primary"
+              :disabled="isDisabled"
+              @click="handleSaveAndContinue"
+            >
+              Save and Continue
+            </v-btn>
+            <v-btn v-if="wizardStore.step === 7" type="submit" :form="getFormId" rounded="lg" color="primary" :disabled="isDisabled" @click="handleSubmit">
```

Comment:
Same here, you can write `wizardStore.currentStepStage === "Review"` and TS should enforce you have a valid PortalStage enum there.